core/src/css/palettes/dark.always.scss

```diff
@@ -1,13 +1,13 @@
 @import "./dark";
 
-:root {
+:root:not(.ionic) {
```

Comment:
Since `core.scss` targets a specific element, this change is needed to apply the correct styles from palettes.

Reply:
I don't think adding `:not(.ionic)` is the right solution. This would force users to target `:root:not(.ionic) {` to override all variables as recommended here: https://ionicframework.com/docs/theming/css-variables#setting-values

See the `src/theme/variables.css` file under this demo: https://ionicframework.com/docs/theming/dark-mode#always

Reply:
I  reverted and went with a different solution: https://github.com/ionic-team/ionic-framework/pull/29707